graph/import.go

```diff
@@ -47,7 +52,21 @@ func (s *TagStore) CmdImport(job *engine.Job) engine.Status {
 		defer progressReader.Close()
 		archive = progressReader
 	}
-	img, err := s.graph.Create(archive, "", "", "Imported from "+src, "", nil, nil)
+
+	buildConfigJob := job.Eng.Job("build_config")
+	buildConfigJob.Stdout.Add(stdoutBuffer)
```

Comment:
@rhatdan Could we do a `r, w := io.Pipe()` and send the writer to stdout, and call `json.NewDecoder(r)` to avoid additional buffers?